src/uct/ib/dc/dc_mlx5_ep.h

```diff
@@ -479,6 +495,7 @@ uct_dc_mlx5_iface_dci_get(uct_dc_mlx5_iface_t *iface, uct_dc_mlx5_ep_t *ep)
     ucs_assert(!iface->super.super.config.tx_moderation);
 
     if (uct_dc_mlx5_iface_is_dci_rand(iface)) {
+        ucs_assert(ep->dci != UCT_DC_MLX5_EP_NO_DCI);
```

Comment:
why needed?

Reply:
updated: this was a fix for Coverity issue (false-positive):
https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=19271&view=logs&j=42a93636-9ba0-5067-13f1-66c03328bbdd&t=156e1e98-827e-5d32-6ea8-39712540f036&l=2524
https://dev.azure.com/ucfconsort/0b36e3f0-8ab9-4a48-b68b-4b2350e02c88/_apis/build/builds/19271/logs/184